doc/kotlin_support.md

```diff
@@ -4,12 +4,32 @@ From 3.0.0 we started support `.kt` file generation.
 
 Let's think about the same example as described in `README.md`.
 
-### 0. Prepare AndroidManifest
+### 0. Preparation
 
 Add the following line to `AndroidManifest.xml`:
  
 `<uses-permission android:name="android.permission.CAMERA" />`
 
+Include the following in your **app module** `build.gradle` file:
+
+`${latest.version}` is [![Download](https://api.bintray.com/packages/hotchemi/maven/permissionsdispatcher/images/download.svg)](https://bintray.com/hotchemi/maven/permissionsdispatcher/_latestVersion)
+
+```groovy
+apply plugin: 'kotlin-kapt'
+
+kapt {
+    generateStubs = true
```

Comment:
I guess this is no longer needed. This is for kapt2 but the latest ver is 3.
https://kotlinlang.org/docs/reference/kapt.html